cmd/run.go

```diff
@@ -501,6 +502,20 @@ func setupExecutors(s *scheduler.Scheduler, config *config.Config, conn *connect
 		mainExecutors[pipeline.AssetTypeIngestr][scheduler.TaskInstanceTypeCustomCheck] = ingestrCustomCheckRunner
 	}
 
+	if s.WillRunTaskOfType(pipeline.AssetTypeAthenaQuery) || estimateCustomCheckType == pipeline.AssetTypeAthenaQuery {
+		athenaOperator := athena.NewBasicOperator(conn, wholeFileExtractor, athena.NewMaterializer(fullRefresh))
```

Comment:
how come you use a regular `conn` object, since it doesn't return a DB instance for athena?

Reply:
I imagine this comment goes together with https://github.com/bruin-data/bruin/pull/71#discussion_r1728723693